### ISSUE TITLE
Add a linter to the project

### ISSUE THREAD

**Issue by mojavelinux** [tj/commander.js#754]
I'm trying to contribute to this project and I have no idea what code style I'm supposed to be using. And there's nothing to check that I'm doing it right. A linter would make contributing to this project a lot easier.

I also find the use of commas to be rather eclectic, so it would be nice to either enforce that style or use a more standard style.

My proposal is to use [standardJS](https://standardjs.com/) with eslint. It's extremely popular and effective. I've used it for several months now and I hardly ever having to think about it.

I'm happy to set up this tooling.

**Comment by abetomo** [comment]
I also like the `standard`.

**Comment by mojavelinux** [comment]
I went ahead and added a linter configuration based on StandardJS with some customizations that adapt it to this project's style.

I left the "yoda" rule enabled and updated the source accordingly. If you disagree, I can roll that back and disable that rule.

I disabled the eqeqeq rule, though I think we should move forward with enabling it.

I changed the comma-leading lines since I couldn't find a rule that allowed us to keep that style.

I'm not running the linter on the tests since we plan to rewrite them anyway. See #649.
